Claudia/src/main/java/com/endava/task/second/Item.java

```diff
@@ -0,0 +1,24 @@
+package com.endava.task.second;
+
+import java.util.Random;
+
+public class Item {
+    public long id = 1;
```

Comment:
You don't need to initialize id here as you initialize it as random value in constructor.
Fix this in all other classes as well.

Reply:
Done